sda-commons-server-kafka/README.md

```diff
@@ -257,6 +257,17 @@ kafka:
       pollInterval: 200
 ```
 
+If the KafkaBundle is not essential for the functionality of your service you can disable the health check manually.
+This way your service can stay healthy even if the connection to Kafka is disrupted.
```

Comment:
It is possible to describe "not essential" a bit further? Can we give some usage example were a service should have a health check and where not?

Reply:
A specific example in our world is the document-access-service. Its cache is updated via kafka, but it could still fulfill its main task (handling requests from the kundenapp and delivering documents to it) without kafka, just with a potentially outdated cache.

Do you already have some general usage examples in mind? Im not sure if we can give usage examples in the documentation as its probably always very specific to the use case.

Reply:
Yeah such a example, a bit more generic. But also an example when the health check is useful, maybe a use-case were the data has to be up-2-date.

Maybe also an example for a producer: A producer might do error handling if the kafka broker is unavailable, e.g. by queuing stuff or telling the user that he can't process data right now. If the producer isn't doing that it might be useful if the health check fails.